typhon/display.py

```diff
@@ -153,7 +153,11 @@ def load_template(self, macros=None):
                                         macros=macros)
             # Add device to all children widgets
             if self.devices:
-                for widget in self._main_widget.findChildren(TyphonBase):
+                designer = (self._main_widget.findChildren(TyphonDesignerMixin)
+                            or [])
+                bases = (self._main_widget.findChildren(TyphonBase)
+                         or [])
+                for widget in set(bases + designer):
```

Comment:
This is OK assuming pyqt, as `findChildren` is defined to return a `list`. I'm not sure about pyside (or if we care about it), as it says: 
![image](https://user-images.githubusercontent.com/5139267/53975230-38591380-40b9-11e9-9493-514b31c91cbf.png)

Reply:
I don't think we are promising support for `PySide` users.